google/cloud/client.py

```diff
@@ -106,7 +106,7 @@ def from_service_account_json(cls, json_credentials_path, *args, **kwargs):
         with io.open(json_credentials_path, "r", encoding="utf-8") as json_fi:
             credentials_info = json.load(json_fi)
 
-        return cls.from_service_account_info(credentials_info)
+        return cls.from_service_account_info(credentials_info, **kwargs)
```

Comment:
It looks like `*args` should probably also be passed here.

Reply:
Nice catch!  Fixed in 6b0b42c, including adding a test for posargs in the `info` case.